tests/framework/microvm.py

```diff
@@ -255,6 +255,18 @@ def kill(self):
                 LOG.exception("Process not found: %d", self.firecracker_pid)
             except FileNotFoundError:
                 LOG.exception("PID file not found")
+
+            # if microvm was spawned then check if it gets killed
+            if self._spawned:
+                # filter ps results for the jailer's unique id
+                rc, stdout, stderr = utils.run_cmd(
+                    f"ps aux | grep {self.jailer.jailer_id}"
+                )
+                assert rc == 0 and stderr == ""
```

Comment:
would it make sense to combine both assertions under the same error message? It looks like if grep finds nothing, it won't return 0.

Reply:
When I tested locally I see run_cmd always returns 0 and stdout has at least 1 grep command
```
ipdb> rc, stdout, stderr = utils.run_cmd(f"ps aux | grep something")
ipdb> rc
0
ipdb> print(stdout)
root          93  0.0  0.0   2892   984 pts/1    S+   10:27   0:00 /bin/sh -c ps aux | grep something
root          95  0.0  0.0   4024  2036 pts/1    S+   10:27   0:00 grep something
```

only if the jailer id is empty, the run_cmd throws an exception as below:
```
ipdb> rc, stdout, stderr = utils.run_cmd(f"ps aux | grep ")
*** ChildProcessError: 
[96] Command:
ps aux | grep 
[96] stdout:

[96] stderr:
Usage: grep [OPTION]... PATTERNS [FILE]...
Try 'grep --help' for more information.

Returned error code: 2
stderr:
Usage: grep [OPTION]... PATTERNS [FILE]...
Try 'grep --help' for more information.
```

Reply:
done.